src/components/TextToSpeech.tsx

```diff
@@ -0,0 +1,55 @@
+import { FC, ReactElement, useEffect, useState, ReactNode } from "react";
+import { Box, Button, Typography, TypographyProps } from "@mui/material";
+import PlayArrowIcon from '@mui/icons-material/PlayArrow';
+import { SxProps, SystemProps } from '@mui/system';
+
+
+
+export const TextToSpeech: FC<TypographyProps> = (props) => {
```

Comment:
Then switch out your props
```suggestion
export const TextToSpeech: FC<TextToSpeechProps> = (props) => {
```

Reply:
decided to only use props passed down by typography